dist/performance.js

```diff
@@ -84,6 +84,66 @@ function getWebVitalsJS() {
     }).map(har => har.url);
 }
 
+function getGamingMetrics(rawDoc) {
+    let returnObj = {};
+    const regexForCheckChromeLH = new RegExp(/.{1}userAgent.{1,100}(?:Chrome-Lighthouse|Google Lighthouse).{2}/)
+    const regexForCheckGTmetrix = new RegExp(/.{1}userAgent.{1,100}(?:GTmetrix|gtmetrix.com).{2}/)
+    const regexForCheckPageSpeed = new RegExp(/.{1}userAgent.{1,100}(?:PageSpeed).{2}/)
+    let scripts = rawDoc.getElementsByTagName('script');
```

Comment:
@25prathamesh this will only look at inline script blocks. We could also look at external scripts in `$WPT_BODIES` if that improves the detection. (type == Script)

Reply:
I avoided it initially because I thought it will add processing overhead to check all script resp bodies. I have committed code changes that will check the external scripts for the user agent check
commit link - https://github.com/HTTPArchive/custom-metrics/commit/7108adbc48a05178263950c6ea433286dbcba741
WPT test - https://www.webpagetest.org/result/220528_AiDcZA_4C1/1/details/#waterfall_view_step1